io.openems.edge.battery.bmw/src/io/openems/edge/battery/bmw/BmwBatteryImpl.java

```diff
@@ -340,7 +337,7 @@ private void setStateMachineState(State state) {
 	}
 
 	@Override
-	protected ModbusProtocol defineModbusProtocol() throws OpenemsException {
+	protected ModbusProtocol defineModbusProtocol() {
```

Comment:
@JohannesHundschell / @pooran-c: this breaks the entire build, because `new ModbusProtocol()` throws an `OpenemsException`. Did you even try to build this project before approving it?